### PR TITLE
add merge_config_parameters scripts

### LINKED ISSUE #19: a paddle_merge_model script is needed
paddle_merge_model  can integrate the model config file and parameter to one files.
It would be very convenient if  there is a script.

### PR DESCRIPTION
**Pull request by NHZlX**
fix #19 
modify readme in pruning
rename `$Mobile/tool` to `$Mobile/tools`

### CODE REVIEW

File: tools/merge_config_paramsters/merge_config_and_params.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+            f.write(conf_str)
+            for pname in param_names:
+                self.params.serialize(pname, f)
+
+if __name__ == '__main__':
+
+    Merge_model().do_merge()
+
+    print 'Generate ', MERGED_FILE
+    print '\nMerge SUCCESS!'
```
Comment (Xreki) [inline tools/merge_config_paramsters/merge_config_and_params.py]: 我倾向于写一个`merge_model.py`放到`https://github.com/PaddlePaddle/Paddle/tree/develop/python/paddle/utils`目录下。`merge_model.py`可定义`def merge_v2_model(net, param_file, out)`这样一个接口，而使用这个工具则需要写这样一个`run.py`：
```python
from paddle.utils.merge_model import merge_v2_model
from mobilenet import mobile_net

if __name__ == '__main__':
    net = mobile_net(3*224*224, #input data size
                     102, #class num (flowers 102)
                     1.0 # mobilenet's scale 
                     )   

    param_file = 'mobilenet_flowers102.tar.gz'
                                                                                                                                                                                             
    # the file we generate
    merged_model = 'mobilenet.paddle'
    merge_v2_model(net, param_file, merged_model)
```

Comment (NHZlX) [reply tools/merge_config_paramsters/merge_config_and_params.py]: perfect

File: tools/merge_config_paramsters/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# import your network configuration
+from mobilenet import mobile_net
+        
+net = mobile_net(3*224*224, 102, 1.0)
+param_file = './mobilenet_flowers102.tar.gz'
+output_file = './output.paddle'
+        
+merge_v2_model(net, param_file, output_file)
+
+```
```
Comment (Xreki) [inline tools/merge_config_paramsters/README.md]: 这个文件里的内容挪到`https://github.com/PaddlePaddle/Mobile/blob/develop/benchmark/tool/C/README.md`这里如何？

Comment (hedaoyuan) [reply tools/merge_config_paramsters/README.md]: 单独放在这里也可以，如何将训练后的多个参数文件合并成一个，这个问题关注的人也挺多的。

Comment (Xreki) [reply tools/merge_config_paramsters/README.md]: OK，那在`https://github.com/PaddlePaddle/Mobile#deployment-optimization-methods`加个链接吧

File: tools/merge_config_paramsters/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**Model Config :** [Mobilenet model config](../../models/mobilenet.py).    
+**Model Parameters:** [Mobilenet model param pretrained on flower102 download](https://pan.baidu.com/s/1geHkrw3) 
+
+### Step2: Merge 
+
+Run the following code
+
+```
+from paddle.util.merge_model import merge_v2_model
```
Comment (Xreki) [inline tools/merge_config_paramsters/README.md]: A typo: from paddle.**utils**.merge_model import merge_v2_model

Comment (NHZlX) [reply tools/merge_config_paramsters/README.md]: ok, i will fix it ASAP

**Comment by Xreki** [comment]
There is another typo in the directory's name: `merge_config_paramsters` -> `merge_config_parameters`

**Review by Xreki — APPROVED** [review]

**Pull request merged**